src/mcnp/mcnp6/patch/mcnp610.patch

```diff
@@ -71,11 +71,11 @@ diff -rN '--unified=0' Source/src/charged_particle_history.F90 Source_dagmc/src/
 +
 +        ! DAGMC: if minimum distance to next event is exactly zero, skip the
 +        ! charged particle physics and go directly to the surface crossing
-+        if( d_step_inter_decay_surf == zero ) goto 100
++        if( d_step_inter_decay_surf == zero ) goto 101
 +
 @@ -627,0 +633,2 @@
 +
-+100     continue  ! DAGMC jump target
++101     continue  ! DAGMC jump target
```

Comment:
Was this a bug?

Reply:
Not a bug. What happened here is that we added a DAGMC jump target and gave it a an ID of 100, but then the MCNP6.2 source code added its own jump target with an ID of 100. So I changed all our patches to use a jump target of 101 instead.